.github/troubleshooting.md

```diff
@@ -50,5 +49,5 @@ The Kiali.io pipeline has a "release" job that in case of errors, the log should
 The manual actions to do in case of a failure are:
 
 * Delete the $RELEASE_VERSION branch (if exists)
-* Revert the last commits made by kiali-bot on the staging branch
```

Comment:
@leandroberetta again, review this line and see if we should say something different? Right now this is being deleted and I don't think it should be deleted.
```
* Revert the last commits made by GitHub Actions on the staging branch
```